BUILD.md

```diff
@@ -36,10 +38,19 @@ make
 ### Run the Coordinator
 
 ```bash
-EDG_COORDINATOR_MESH_ADDR=localhost:2001 EDG_COORDINATOR_CLIENT_ADDR=localhost:4433 EDG_COORDINATOR_DNS_NAMES=localhost EDG_COORDINATOR_SEAL_DIR=$PWD OE_SIMULATION=1 erthost build/coordinator-enclave.signed
+OE_SIMULATION=1 erthost build/coordinator-enclave.signed
 ```
 
-*Note*: the Coordinator's state is sealed to `$PWD/sealed_data`. If you want a fresh restart remove this file first: `rm $PWD/sealed_data`.
+Per default, the Coordinator starts with the following default values. You can set your desired configuration by setting the environment variables.
+
+| Setting | Default Value | Environment Variable |
+| --- | --- | --- |
+| the listener address for the gRPC server | localhost:2001 |  EDG_COORDINATOR_MESH_ADDR |
+| the listener address for the HTTP server | localhost: 4433 | EDG_COORDINATOR_CLIENT_ADDR |
```

Comment:
a bit out of scope because not introduced with this PR but: We should talk about the purpose of the servers rather than their technical implementation